src/app/viam-client.ts

```diff
@@ -0,0 +1,35 @@
+import { grpc } from '@improbable-eng/grpc-web';
+import { type DialOptions } from '@viamrobotics/rpc/src/dial';
+import { DataServiceClient } from '../gen/app/data/v1/data_pb_service';
+import { createViamTransportFactory } from '../robot/dial';
+import { DataClient } from './data-client';
+
+export class ViamClient {
+  private serviceHost: string;
+  private dialOpts: DialOptions;
+  private transportFactory: grpc.TransportFactory | undefined;
+  public dataServiceClient: DataServiceClient | undefined;
+  public dataClient: DataClient | undefined;
+
+  constructor(serviceHost: string, dialOpts: DialOptions) {
```

Comment:
will the service host ever change? should we have a default instead of forcing users to always have to specify a service host?

Reply:
also add a comment on what service host should be

Reply:
Good point. In Python we don't even have a default. They hard coded it to always be `app.viam.com:443`. I will make that change.

Reply:
I would like to see the service host have a default, but continue to allow users to specify a different URL - this will be good for testing